appinfo/info.xml

```diff
@@ -26,7 +26,6 @@ The administrator is then logged-in as the user, to switch back to the regular u
     <author>Nextcloud</author>
     <namespace>Impersonate</namespace>
 
-    <category>tools</category>
```

Comment:
why remove it?

Reply:
ah, bullshit. in 13 it was duplicated, took it from there. messy.